hw/chisel/src/test/scala/snax/streamer/StreamerTopGenerate.scala

```diff
@@ -0,0 +1,16 @@
+package snax.streamer
+
+import chisel3._
+import org.scalatest.flatspec.AnyFlatSpec
+import chiseltest._
+
+class StreamerTopGenerate
+extends AnyFlatSpec {
+
+  emitVerilog(
+    new StreamerTop(TestParameters.streamer),
+    Array("--target-dir", "generated/")
+  )    
+
+}
```

Comment:
Can we add the modular system verilog generation test back? It is useful for modular generation and tests.

Reply:
What is the modular system verilog genration?

Reply:
We do not want people to use subparts as systemverilog components, right?
The testing of systemverilog generation is done by the top module, the only one that we are going to use. All lower modules are included in this test. Having them separate is duplicated testing, and is confusing as we don't want to generate verilog for the modules separately. Does that make sense?

Reply:
Ok, I understand your point of view. I am fine to merge this PR.

Reply:
Ok, thank you!